presto-docs/src/main/sphinx/cache.rst

```diff
@@ -5,4 +5,5 @@ Cache
 .. toctree::
     :maxdepth: 1
 
+    cache/library
```

Comment:
library and alluxio, seems confusing section to me. How about we create section for local and service(remote)?
WDYT @apc999 ?

Reply:
local and service can work